protos/orchestrator_service.proto

```diff
@@ -797,6 +797,10 @@ enum WorkerCapability {
     // When set, the service may return work items without any history events as an optimization.
     // It is strongly recommended that all SDKs support this capability.
     WORKER_CAPABILITY_HISTORY_STREAMING = 1;
+
+    WORKER_CAPABILITY_SCHEDULED_TASKS = 2;
```

Comment:
It's worth adding a short comment describing what this does/what it may depend on. For instance, this one requires entities and the other requires the external blob storage, right?